hasher-matcher-actioner/hmalib/common/actioner_models.py

```diff
@@ -33,3 +38,51 @@ class ActionRule:
     action_label: ActionLabel
     must_have_labels: t.List[Label]
     must_not_have_labels: t.List[Label]
+
+
+TUrl = t.Union[t.Text, bytes]
```

Comment:
There might be a specific type for this buried in the typing module

Reply:
Couldn't find